app/views/admins/articles/index.html.erb

```diff
@@ -42,7 +42,7 @@
           <span class='hover-target', tabindex='0'><%= article.user.email %></span>
           <aside class='hover-popup'><%= article.user.email %></aside>
         </div>
-        <div class="cell"><%= article.status %></div>
+        <div class="cell text-<%= @presenters.find { |presenter| presenter.article == article }.color %>"><%= article.status %></div>
```

Comment:
`ArticlePresenter.new(article).color ` that's it

remove @presenters variable

Reply:
Fixed. Removed redundant presenter's initializations.